iofog-agent-client/build.gradle

```diff
@@ -3,9 +3,15 @@ plugins {
 }
 
 description = 'iofog-agent-client'
-
+configurations {
+    // configuration that holds jars to include in the jar
+    extraLibs
+}
 dependencies {
     compile 'org.slf4j:slf4j-nop:1.7.13'
+	extraLibs ('org.apache.geronimo.specs:geronimo-json_1.0_spec:1.0-alpha-1',
```

Comment:
`https://mvnrepository.com/artifact/javax.json/javax.json-api` contains packages you need

Reply:
updated